crds/operators.coreos.com_catalogsources.yaml

```diff
@@ -90,6 +90,11 @@ spec:
                   a registry-server with. Only used when SourceType = SourceTypeGrpc.
                   If present, the address field is ignored.
                 type: string
+              priority:
+                description: Priority field, if specified, is used to rank catalog
+                  sources that contain operators to supply dependent APIs and prioritize
+                  certain catalogs based on the ranking.
+                type: integer
```

Comment:
This sentence is very hard to understand. Could we try something simpler like

`Description: The priority assigns a weight to the catalog source that is then used during dependency resolution. Higher priorities indicate that this catalog source should be considered earlier in the dependency search.`

Reply:
I have rephrased it

Reply:
Might want to include default priorities, default values, range, and behaviors if priorities are the same. basically a brief paraphrase to 

- The higher the value, the higher the priority. 
- The default priorities for Red Hat, Certified, and Community catalog sources are 10000,5000,1000  respectively. 
- The range of the priority value can go from positive to negative in the range of int32. 
- The default value to a catalog source with unassigned priority would be 0. This means, if cluster admins would like to make sure certain catalogs get considered last, they can assign the catalog with negative priority values. 
- The catalog source with the same priority values will be ranked lexicographically. 
-

Reply:
>The default priorities for Red Hat, Certified, and Community catalog sources are 10000,5000,1000 respectively.

I'd omit this one. Creation of specific catalog sources is a usage of this API, not a semantic of the API itself.

Reply:
+1, that documentation should just live in the downstream component that builds these objects

Reply:
Updated the description

Reply:
The mentions of provided catalogs don't belong here -- provided catalogs are just catalogs. As Kevin said, that belongs in the documentation of the component that creates those catalogs.